modules/SSM_Document_ResizeVolume/fragments/volume.yaml

```
@@ -0,0 +1,67 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Description: AWS CloudFormation template to resize EBS volume.
+Parameters:
+  DocumentName:
+    Type: String
+    Description: The SSM document name. 
+    Default: Resize-EBS-Volume
+  VolumeSize: 
```

Is it useful to have a parameter that only changes the default?


If so, it might be better to be explicit about this in the name
```suggestion
  DefaultVolumeSize: 
```

---

This parameter is used by the caller to set the volume size; is not to be used as the default value alone.

---

I meant the default in the ssm document. That's the only place I see it used ([link](https://github.com/aws-cloudformation/community-registry-extensions/pull/209/files#diff-fc450dd875b65b6d980a1889ecc845ae1797651da7669eb89fdc40dab134236fR27))